c12930501.lua

```diff
@@ -28,7 +28,9 @@ function c12930501.condition(e,tp,eg,ep,ev,re,r,rp)
 	return tp~=Duel.GetTurnPlayer()
 end
 function c12930501.filter(c,tp)
-	return c:IsType(TYPE_SPELL) and c:IsSSetable() and (c:IsType(TYPE_FIELD) or Duel.GetLocationCount(tp,LOCATION_SZONE)>0)
+	local ct=Duel.GetLocationCount(tp,LOCATION_SZONE)
+	if e:IsHasType(EFFECT_TYPE_ACTIVATE) and not e:GetHandler():IsLocation(LOCATION_SZONE) then ct=ct-1 end
```

Comment:
param e is undefined.

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1862/commits/f703b54ec61857e5227f4e8196e1bbf9ff880dd8 fixed